random_question/weighted_random.py

```diff
@@ -10,11 +10,26 @@
 
 # TODO: Hess should implement!
 def weighted_sampler(data, weights):
-    """Should return a random item from data array, weighted by the weights array"""
-    raise NotImplementedError("Hess should implement!")
+    random_number=random.random()
+    print('this is the:')
```

Comment:
To save the second print you can:
`print('this is the: ' + str(random_number))`
or
`print('this is the: %f'.format(random_number))`
but the best way is to
`print('this is the: %f' & random_number)`

please run my suggestions in a python terminal

Reply:
done